app/serializers/costs_parameters_serializer.rb

```diff
@@ -5,46 +5,135 @@
+      costs_of_insulation_apartments_1945_1964
+      costs_of_insulation_apartments_1965_1984
+      costs_of_insulation_apartments_1985_2004
+      costs_of_insulation_apartments_2005_present
+      costs_of_insulation_apartments_future
+      costs_of_insulation_detached_houses_before_1945
+      costs_of_insulation_detached_houses_1945_1964
+      costs_of_insulation_detached_houses_1965_1984
+      costs_of_insulation_detached_houses_1985_2004
+      costs_of_insulation_detached_houses_2005_present
+      costs_of_insulation_detached_houses_future
+      costs_of_insulation_semi_detached_houses_before_1945
+      costs_of_insulation_semi_detached_houses_1945_1964
+      costs_of_insulation_semi_detached_houses_1965_1984
+      costs_of_insulation_semi_detached_houses_1985_2004
+      costs_of_insulation_semi_detached_houses_2005_present
+      costs_of_insulation_semi_detached_houses_future
+      costs_of_insulation_terraced_houses_before_1945
+      costs_of_insulation_terraced_houses_1945_1964
+      costs_of_insulation_terraced_houses_1965_1984
+      costs_of_insulation_terraced_houses_1985_2004
+      costs_of_insulation_terraced_houses_2005_present
+      costs_of_insulation_terraced_houses_future
+      households_ht_heat_delivery_system
+      households_mt_heat_delivery_system
+      households_lt_heat_delivery_system
     ],
     costs_building_and_installations_buildings: %w[
-      costs_building_and_installations_buildings_insulation
-      costs_building_and_installations_buildings_heat_delivery_system
+      costs_of_insulation_buildings_present
+      costs_of_insulation_buildings_future
+      buildings_ht_heat_delivery_system
+      buildings_mt_heat_delivery_system
+      buildings_lt_heat_delivery_system
     ],
     costs_storage_and_conversion_storage: %w[
       costs_hydrogen_storage
-      heat_infrastructure_ht_storage_annualised_costs
-      heat_infrastructure_mt_storage_annualised_costs
-      heat_infrastructure_lt_storage_annualised_costs
+      heat_infrastructure_ht_storage
+      heat_infrastructure_mt_storage
+      heat_infrastructure_lt_storage
     ],
-    costs_infrastructure_electricity: %w[
-      total_costs_of_electricity_network_calculation
+    costs_carriers_biomass: %w[
+      costs_carriers_biomass
     ],
-    costs_infrastructure_heat: %w[
-      costs_infrastructure_heat
+    costs_carriers_oil_and_products: %w[
+      costs_carriers_oil_and_products
     ],
-    costs_infrastructure_hydrogen: %w[
-      
+    costs_carriers_coal_and_products: %w[
+      costs_carriers_coal_and_products
     ],
-    costs_infrastructure_network_gas: %w[
-      gas_network_total_costs
+    costs_carriers_uranium: %w[
+      costs_carriers_uranium
+    ],
+    costs_carriers_heat: %w[
+      costs_carriers_heat
+    ],
+    costs_carriers_natural_gas: %w[
+      costs_carriers_natural_gas
+    ],
+    costs_carriers_waste: %w[
+      costs_carriers_waste
     ],
```

Comment:
Is there a reason why all these carriers have a separate group, instead of putting under the same generic group `costs_carriers`.